Send JSON Webhooks in campaigns when content-type of application/json is included in the header (#101)
mautic/mautic-documentation: pages/05.Campaigns/03.Campaign Events/docs.en.md

Mautic 2.15.0 [adds][215-ip-as-token] possibility to use contact's IP address as a token `{contactfield=ipAddress}`.

> Mautic 3.1.0 [adds][8959-send-content-as-json] possibility to send webhook payload in JSON format, to send JSON payload you have to add to headers an entry with label `content-type` and value `application/json`

Reviewer: ```suggestion
Mautic 3.1.0 [adds][8959-send-content-as-json] the possibility to send webhook payloads in JSON format.  To send a JSON payload, add a header entry with the label `content-type` and value `application/json`:
```
